common/app/views/fragments/commercial/badge.scala.html

```diff
@@ -1,6 +1,7 @@
 @(content: model.ContentType)(implicit request: RequestHeader)
 @import common.Edition
 @import conf.switches.Switches.SponsoredSwitch
+@import conf.switches.Switches.NewCommercialContent
```

Comment:
Shouldn't you import this switch rather than the whole switch collection in the previous changes too?